deps/libbpf/ci/vmtest/run_selftests.sh

```
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source $(cd $(dirname $0) && pwd)/helpers.sh
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)